src/app/header/header.component.ts

```diff
@@ -14,12 +14,6 @@ import { Navigation } from './../models/navigation';
 import { MenuItem } from './../models/menu-item';
 import { DummyService } from './../shared/dummy.service';
 
-
-
-interface MenuHiddenCallback {
```

Comment:
You can't remove this, it breaks the build. Or else you need to find all the places that call it and change them.